src/averageAges.js

```diff
@@ -37,7 +44,17 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter((person) => person.sex === 'f');
```

Comment:
You can simplify the women filtering by using a single filter statement with the && operator instead of using ternary operator.